content/docs/tooling.md

```diff
@@ -41,6 +41,7 @@ The following is a list of tools that validate AsyncAPI documents.
 | Link           | Description    |
 | :------------- | :------------- |
 | [Check-API](https://github.com/Mermade/check_api) | It allows you to validate a local file or remote URL with a single command-line or programmatic invocation. It returns an exitCode of 0 on success and 1 on failure, making it suitable for use in Continuous Integration environments. Thanks to [@PermittedSoc](https://twitter.com/@Permittedsoc).
+| [asyncapi-validator](https://github.com/WaleedAshraf/asyncapi-validator) | It allows you to validate schema of your messages against your AsyncAPI schema defination. You can use it with Kafka, RabbitMQ or any other messaging/queue. Thanks to [@waleedashraf](https://twitter.com/@waleedashraf01).
```

Comment:
> Thanks to [@waleedashraf](https://twitter.com/@waleedashraf01).

Are you thanking yourself? :trollface:

Reply:
hahaha :D :D I was just following the template.